app/network/Shared/Resources/Localizable.xcstrings

```diff
@@ -11679,21 +11679,21 @@
             "state" : "translated",
             "value" : "一般"
           }
+        },
+        "cs" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Obecné"
+          }
         }
       }
     },
-    "Get in touch" : {
-      "localizations" : {
-        "ar" : {
```

Comment:
Also, not sure if this is intentional, we're dropping the Arabic translation here. Could be related.